lib/kuzzle/event/kuzzleEventEmitter.ts

```diff
@@ -45,7 +49,15 @@ class PluginPipeDefinition {
   }
 }
 
-class KuzzleEventEmitter extends EventEmitter {
+export class KuzzleEventEmitter extends EventEmitter {
+  public superEmit: any;
+  public pipeRunner: any;
+  public pluginPipes: any;
+  public pluginPipeDefinitions: any;
+  //eslint-disable-next-line @typescript-eslint/ban-types
+  public corePipes: Map<string, Function[]>;
+  public coreAnswerers: any;
+  public coreSyncedAnswerers: any;
```

Comment:
I suggest to modify types like : 

```
  public superEmit: any;
  public pipeRunner: PipeRunner;
  public pluginPipes: Map<any, any>;
  public pluginPipeDefinitions: Map<any, PluginPipeDefinition>;
  //eslint-disable-next-line @typescript-eslint/ban-types
  public corePipes: Map<string, Function[]>;
  public coreAnswerers: Map<any, any>;
  public coreSyncedAnswerers: Map<any, any>;
```

maybe you can replace more any too

Reply:
Done (best effort to find real Class)